support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -213,9 +215,48 @@ function WeeklyCheckoutForm(props: PropTypes) {
             />
           </FormSection>
           <FormSection title="Where should we deliver your magazine?">
-            <DeliveryAddress />
+            {props.billingPeriod !== 'SixWeekly' ?
+              <CheckboxInput
+                text="This is a gift"
+                checked={props.orderIsAGift}
+                onChange={() => props.setGiftStatus(!props.orderIsAGift)}
+              />
+            : null}
+            {!props.orderIsAGift ? <DeliveryAddress /> : null}
           </FormSection>
-          <FormSection title="Is the billing address the same as the delivery address?">
+          {props.orderIsAGift ? (
+            <FormSection title="Gift recipient's details">
+              <SelectWithLabel
+                id="title"
+                label="Title"
+                optional
+                value={props.titleGiftRecipient}
+                setValue={props.setTitleGift}
+              >
+                <option value="">--</option>
+                {options(titles)}
+              </SelectWithLabel>
+              <PersonalDetailsGift
+                firstName={props.firstNameGiftRecipient}
+                setFirstName={props.setFirstNameGift}
+                lastName={props.lastNameGiftRecipient}
+                setLastName={props.setLastNameGift}
+                email={props.emailGiftRecipient}
+                setEmailGift={props.setEmailGift}
+                formErrors={((props.formErrors: any): FormError<PersonalDetailsFormField>[])}
+                isGiftRecipient
+              />
+            </FormSection>)
+          : null}
+          {props.orderIsAGift ? (
```

Comment:
I don't think you need another conditional check here do you? you can just include this in the one above.

Reply:
Very true. Thanks Rupert!